src/Explorer/Tabs/Tabs.tsx

```diff
@@ -47,15 +49,20 @@ export const Tabs = ({ explorer }: TabsProps): JSX.Element => {
         <div className="nav-tabs-margin">
           <ul className="nav nav-tabs level navTabHeight" id="navTabs" role="tablist">
             {openedReactTabs.map((tab) => (
-              <TabNav key={ReactTabKind[tab]} active={activeReactTab === tab} tabKind={tab} />
+              <TabNav
+                key={ReactTabKind[tab]}
+                active={activeReactTab === tab}
+                tabKind={tab}
+                queryCopilotState={queryCopilotState}
+              />
             ))}
             {openedTabs.map((tab) => (
-              <TabNav key={tab.tabId} tab={tab} active={activeTab === tab} />
+              <TabNav key={tab.tabId} tab={tab} active={activeTab === tab} queryCopilotState={queryCopilotState} />
```

Comment:
You don't need to pass the state to the `TabNav` component. The only thing you need from it is the `resetQueryCopilotStates` function, which you can call directly by doing: `useQueryCopilotState.getState().resetQueryCopilotStates()`

Reply:
Same as above, I cannot call getState() method.